sdk/python/feast/client.py

```diff
@@ -1001,7 +1002,12 @@ def get_historical_features(
             )
         else:
             return start_historical_feature_retrieval_job(
-                self, entity_source, feature_tables, output_format, output_location,
+                client=self,
+                project=self.project or FEAST_DEFAULT_OPTIONS[CONFIG_PROJECT_KEY],
```

Comment:
By the way, shouldn't `self.project` be returning a default if it doesn't exist?

Reply:
yep, it actually does. Will delete this